Global.props

```diff
@@ -96,7 +96,7 @@
         <NugetVersionFilePath>$(MSBuildThisFileDirectory).nugetVersion</NugetVersionFilePath>
         <BuildNugetVersion Condition="Exists($(NugetVersionFilePath))">$([System.IO.File]::ReadAllText($(NugetVersionFilePath)))</BuildNugetVersion>
 
-        <CoreSdkVersion>2.2.0-beta2-build31642</CoreSdkVersion>
+        <CoreSdkVersion>2.2.0-beta3-build34090</CoreSdkVersion>
```

Comment:
We always have a specific version. When we build public versions - they wouldn't have - `builldFoo` suffix